seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -310,7 +409,8 @@
         "<blockquote>.blue-text {<br>&nbsp;&nbsp;color: blue;<br>}</blockquote>",
```

Comment:
Sorry for not being clearer on this. Since the challenge before introduces both how to write a class in CSS and how to apply it, I think the description here should be as simple as:
>Classes allow you to use the same CSS styles on multiple HTML elements. You can see this by applying your `red-text` class to the `p` element.